src/inputs/Table/components/TableToolbar/TableToolbar.js

```diff
@@ -38,7 +39,7 @@ export const TableToolbar = (props) => {
         <DeleteRowButton
           onDeleteRow={onDeleteRow}
           labels={labels.deleteRows}
-          disabled={!editMode || !isReady || isLoading}
+          disabled={!editMode || !isReady || isLoading || !(isRowSelected ?? true)}
```

Comment:
There is a problem with `isRowSelected !== true`. If `isRowSelected` is _null_ or _undefined_ (if the integrator has not passed `isRowSelected` as props), it result that _disabled_ will be set to true and it would be more logic that prop absence doesn't disabled the DeleteRowButton.

Maybe, we can set isRowSelected prop default value as _false_ otherwise, more simply, we can pass a noRowSelected as prop.